server/notification-providers/smspartner.js

```diff
@@ -0,0 +1,48 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class SMSPartner extends NotificationProvider {
+    name = "SMSPartner";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://api.smspartner.fr/v1/send";
+
+        try {
+            //Lets remove non ascii char
+            let cleanMsg = msg.replace(/[^\x00-\x7F]/g, "");
```

Comment:
Lets do all `msg`-saintisation here.

```suggestion
            // smspartner does not support non ascii characters and only a maximum 639 characters
            let cleanMsg = msg.replace(/[^\x00-\x7F]/g, "").substring(0, 639);
```

Reply:
Thanks, it's much clearer this way.